### DISCUSSION TITLE
idea

### DISCUSSION THREAD

**Discussion by leonuo** [teaxyz/white-paper#35] [Ideas]
I would like to see a separate division of companies that give their developers a tip for the work done.

**Comment by mxcl** [comment]
Companies will be able to steep tea towards the open source they use. This will be transparent. Companies that do not support their open source will likely face repercussions from the community and package authors they profit from.

tea will not itself name and shame anyone.

**Comment by AmirGhMohseni** [comment]
Can you use AI tools for tea? Changing trading offers by AI

**Reply by harismanciripto111** [reply]
this is a great idea

**Comment by thomas-borrel** [comment]
Thanks - Closing.
